bitcoin/src/p2p/address.rs

```diff
@@ -308,7 +308,7 @@ mod test {
 
     use hex::{test_hex_unwrap as hex, FromHex};
 
-    use super::{AddrV2, AddrV2Message, Address};
+    use super::*;
```

Comment:
Bother, thanks - I'll re-look.

Reply:
You might need to compile with nightly, and possibly nightly from a week ago, because I believe they just reverted a bunch of the "redundant import" lints due to mass protests.